### ISSUE TITLE
do we assume y_pred of keras.losses.categorical_crossentropy(y_true, y_pred) comes from softmax layer?

### ISSUE THREAD

**Issue by Crysitna** [keras-team/keras#12573]
according to the [tensorflow documentation](https://www.tensorflow.org/api_docs/python/tf/keras/backend/categorical_crossentropy), it seems y_preds are assumed to come from softmax when from_logits=False, yet how about keras? there is no such a parameter from_logits in keras.losses.categorical_crossentropy()

And also, if it's assumed to come from softmax, isn't the name categorical_crossentropy misleading since it sounds like combined softmax and CE ?

**Comment by tranvohuy** [comment]
From [keras code](https://github.com/keras-team/keras/blob/766e5f7a6e95d46ba9530c470dc19468a20bd711/keras/losses.py#L68), there is no parameter 'from_logits'
